core/launcher/args/api_container_args.go

```diff
@@ -47,8 +48,16 @@ type APIContainerArgs struct {
 
 	//If its running in a CI environment
 	IsCI bool `json:"is_ci"`
+
+	// The Cloud User ID of the current user if available
+	CloudUserID metrics_client.CloudUserID `json:"cloud_user_id"`
+
+	// The Cloud Instance ID of the current user if available
+	CloudInstanceID metrics_client.CloudInstanceID `json:"cloud_instance_id"`
 }
 
+var skipValidation = []string{"cloud_user_id", "cloud_instance_id"}
```

Comment:
nit: you could use a set (map[string]bool) instead and avoid having to iterate the slice